src/linearProgress/LinearProgress.tsx

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import { View, StyleProp, ViewStyle, Animated, Platform } from 'react-native';
+import { withTheme, FullTheme } from '../config';
+import Color from 'color';
+
+export type LinearProgressProps = {
+  value?: number; // 0 to 1
+  variant?: 'determinate' | 'indeterminate';
+  color?: 'primary' | 'secondary';
```

Comment:
Color being limited to only the primary or secondary color seems overly restrictive. People will want to be able to style this however they want.

Reply:
Perhaps just default to the secondary color otherwise allow people to override the color.

Reply:
Sure Sir! Will do that